.github/workflows/create-update-issues.yaml

```
@@ -0,0 +1,45 @@
+name: Create Update Issues
+
+on:
+  workflow_call:
+    secrets:
+      GH_TOKEN:
+        description: Github token with access to mobile and extention repositories
```

This adds more specifics on the required permissions, and fixes a typo:
```suggestion
        description: Github token with permission to create issues in both mobile and extension repositories
```